specs/scoring_spec.rb

```diff
@@ -42,21 +43,33 @@
 
   describe 'highest_score_from' do
     it 'returns nil if no words were passed' do
+      empty_array = []
+      Scrabble::Scoring.highest_score_from(empty_array).must_be_nil
     end
 
     it 'returns the only word in a length-1 array' do
+      one_word_array = ["apple"]
+      Scrabble::Scoring.highest_score_from(one_word_array).must_equal("apple")
     end
 
     it 'returns the highest word if there are two words' do
+      two_word_array = ["apple", "cat"] #apple = 9; cat = 5
+      Scrabble::Scoring.highest_score_from(two_word_array).must_equal("apple")
     end
 
     it 'if tied, prefer a word with 7 letters' do
+      tie_array = ["lalalal", "foul"] #both 7 value
+      Scrabble::Scoring.highest_score_from(tie_array).must_equal("lalalal")
```

Comment:
Since `lalalal` gets the 7-letter bonus, I don't think you actually have a tie here. There isn't a pair of English words that allow you to test this logic, but `zzzzzz` and `oratory` will do.

"Why bother" you ask? Well, if Miriam-Webster adds `zzzzzz` as an onomatopoeia for snoring, we need to be prepared, or you could envision a scrabble app that works across languages or that allows users to add their own words. Moreover, maybe this is the mathematician in me speaking, but I believe there's beauty and joy to be found in writing code that isn't just "good enough", but which truly solves the problem.